package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jquery-validation-unobtrusive",
-  "version": "3.2.12",
+  "version": "3.2.13",
```

Comment:
Should the versions (here and in version.props) change more given the major dev dependency bumps and minor regular dependency bumps❔ I realize the dev dependency changes shouldn't affect most consumers but thought I'd ask&hellip;

Reply:
Well, all those are internal dependencies for our library, whereas the API of our library didn't change. That said, I plan to bump up the version in a separate commit after all this is settled, mostly for the license update, which can be considered as a non-behavior breaking change.